textproc/chordii/Portfile

```diff
@@ -0,0 +1,22 @@
+PortSystem          1.0
+
+name                chordii
+version             4.5.3
+
+categories          textproc
+maintainers         breun.nl:nils openmaintainer
+platforms           darwin
+license             GPL-2
+description         Produce a professional looking PostScript sheet-music from an ASCII file containing lyrics and chords information.
+long_description    chordii produces a PostScript document from a lyrics file \
+                    containing chord indications and chorus delimiters. The \
+                    document produced contains the lyrics of a song, with the \
+                    guitar chords appearing above the right words. A \
+                    representation of all chords used in the song is printed \
+                    at the bottom of the last page.
+
+homepage            https://www.vromans.org/johan/projects/Chordii/
+master_sites        https://sourceforge.net/projects/chordii/files/chordii/4.5/
```

Comment:
Usually this is done as:
```tcl
set version_major   [join [lrange [split ${version} .] 0 1] .]
master_sites        sourceforge:${name}/${version_major}/
```
or perhaps
```tcl
master_sites        sourceforge:project/${name}/${name}/${version_major}/
```
(not sure about the difference & preference)

The first weird line is there to convert `4.5.3` to `4.5`, so that you don't have to correct at two places next time when you do the upgrade. The second line used to be just sourceforge (and thus greatly simplified), but then there were some redirects involved and I'm not sure how's the situation now.